receiver/windowsservicereceiver/documentation.md

```diff
@@ -0,0 +1,31 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# windowsservice
+
+## Default Metrics
+
+The following metrics are emitted by default. Each of them can be disabled by applying the following configuration:
+
+```yaml
+metrics:
+  <metric_name>:
+    enabled: false
+```
+
+### windows.service
+
+Gauge value containing service status as an integer value.
+
+Gauge values map to service status as follows: 0 - Failed to retrieve service status, 1 - Stopped, 2 - Start Pending, 3 - Stop Pending, 4 - Service Running, 5 - Continue Pending, 6 - Pause Pending, 7 - Service Paused
```

Comment:
nit: it will be nice to have this as a list with each value and meaning on a single line, or perhaps row in a table.